### PR TITLE
migrate rekor docs

### PR DESCRIPTION
**Pull request by endorama**
Migrate Rekor documentation from the repo to the website.

Closes https://github.com/sigstore/rekor/issues/444

### CODE REVIEW

File: content/en/rekor/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Getting Started"
```
Comment (lukehinds) [inline content/en/rekor/getting-started.md:2]: I would drop this if it does not exist yet

File: content/en/rekor/release-log.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Release log"
 category: "Rekor"
-position: 6
+position: 331
```
Comment (lukehinds) [inline content/en/rekor/release-log.md:4]: same for this one

**Comment by endorama** [comment]
@lukehinds remove both pages.

**Comment by endorama** [comment]
@lukehinds Will do ASAP.

**Review by lukehinds — APPROVED** [review]
Thanks @endorama !

**Pull request merged**